### DISCUSSION TITLE
Handling uploading files on localization

### DISCUSSION THREAD

**Discussion by Misagh90** [livewire/livewire#6525] [1. Help]
I tried the [setUpdateRoute()](https://livewire.laravel.com/docs/upgrading#localization) for uploading files to my multilingual website but still have problem (404 error) to handling files in livewire, I asked it in my last [discussion](https://github.com/livewire/livewire/discussions/6287)
Is there anyone who can tell me exactly what to do?! I'm not a professional !

**Comment by joshhanley** [comment]
@Misagh90 what does your routes file look like?

**Reply by Misagh90** [reply]
my routes:
```
Route::group(['prefix' => LaravelLocalization::setLocale(),'middleware' => [ 'localize','localeSessionRedirect', 'localizationRedirect', 'localeViewPath' ]], function () {
    Route::get('/', [App\Http\Controllers\Front\HomeController::class, 'main'])->name('home');
    Route::get(LaravelLocalization::transRoute('routes.about'), [App\Http\Controllers\Front\AboutController::class, 'main'])->name('about');
    Route::get(LaravelLocalization::transRoute('routes.contact'), [App\Http\Controllers\Front\ContactController::class, 'main'])->name('contact');
    Livewire::setUpdateRoute(function ($handle) {
        return Route::post('/livewire/update', $handle);
    });
});
```

& my lang middleware:
```
    public function handle(Request $request, Closure $next): Response
    {
        if (!in_array($request->segment(1),config('translatable.locales'))) {
            $segments = $request->segments();
            $segments = \Arr::prepend($segments, config('app.fallback_locale'));
            return redirect()->to(implode('/', $segments));
        }
        return $next($request);
    }
```

**Reply by joshhanley** [reply]
@Misagh90 thanks! Can you share a screenshot what the network request looks like in your devtools?

**Reply by Misagh90** [reply]
@joshhanley  Sure
![Screenshot (840)](https://github.com/livewire/livewire/assets/6697222/d40f9c22-36df-454c-bd05-f23e76c896fe)

![Screenshot (841)](https://github.com/livewire/livewire/assets/6697222/3de8e741-7181-47fe-a62d-878c3a8d63f1)

![Screenshot (842)](https://github.com/livewire/livewire/assets/6697222/df4a9eb0-e001-44fe-99fe-7c2704b84a28)
